doc/man7/EVP_KDF-PBKDF2.pod

```diff
@@ -60,6 +60,14 @@ and the FIPS provider uses a default mode of 0.
 
 The value string is expected to be a decimal number 0 or 1.
```

Comment:
This should probably say something related to FIPS compliance here.

Reply:
Updated